backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -476,28 +479,78 @@ public UUID getPermissibleOrgId(UUID orgId) {
+  private List<ApiUser> getOrgAdminUsers(UUID orgId) {
     Organization org =
         organizationRepository.findById(orgId).orElseThrow(NonexistentOrgException::new);
-    List<String> adminUserEmails = oktaRepository.fetchAdminUserEmail(org);
-
-    return adminUserEmails.stream()
-        .map(
-            email -> {
-              Optional<ApiUser> foundUser = apiUserRepository.findByLoginEmail(email);
-              if (foundUser.isEmpty()) {
-                log.warn(
-                    "Query for admin users in organization "
-                        + orgId
-                        + " found a user in Okta but not in the database. Skipping...");
-              }
-              return foundUser.map(user -> user.getInternalId()).orElse(null);
-            })
-        .filter(Objects::nonNull)
+    try {
+      List<String> adminUserEmails = oktaRepository.fetchAdminUserEmail(org);
+      return adminUserEmails.stream()
+          .map(
+              adminUserEmail -> {
+                Optional<ApiUser> foundUser = apiUserRepository.findByLoginEmail(adminUserEmail);
+                if (foundUser.isEmpty()) {
+                  log.warn(
+                      "Query for admin users in organization "
+                          + org.getInternalId()
+                          + " found a user in Okta but not in the database. Skipping...");
+                }
+                return foundUser.orElse(null);
+              })
+          .filter(Objects::nonNull)
+          .collect(Collectors.toList());
+    } catch (IllegalGraphqlArgumentException e) {
+      return List.of();
+    }
+  }
+
+  private List<String> getOrgAdminUserEmails(UUID orgId) {
+    return getOrgAdminUsers(orgId).stream()
+        .map(ApiUser::getLoginEmail)
+        .collect(Collectors.toList());
+  }
+
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public List<UUID> getOrgAdminUserIds(UUID orgId) {
+    return getOrgAdminUsers(orgId).stream()
+        .map(ApiUser::getInternalId)
+        .collect(Collectors.toList());
+  }
+
+  private List<String> getOrgAdminEmailsByFacilityState(String state) {
+    List<Facility> facilitiesInState = facilityRepository.findByFacilityState(state);
+    return facilitiesInState.stream()
+        .map(f -> f.getOrganization().getInternalId())
+        .distinct()
+        .map(this::getOrgAdminUserEmails)
+        .flatMap(List::stream)
+        .sorted()
         .collect(Collectors.toList());
   }
 
+  private List<String> getOrgAdminEmailsByPatientState(String state) {
+    List<Organization> orgs = organizationRepository.findAllByPatientStateWithTestEvents(state);
+    return orgs.stream()
+        .map(o -> o.getInternalId())
+        .map(this::getOrgAdminUserEmails)
+        .flatMap(List::stream)
+        .sorted()
+        .collect(Collectors.toList());
+  }
+
+  @AuthorizationConfiguration.RequireGlobalAdminUser
+  public Integer sendOrgAdminEmailCSV(String type, String state) {
```

Comment:
❓ Is there any special logging we want to do in this method for audit logging purposes? 🤔